test/workers/sphinx_indexation_worker_test.rb

```diff
@@ -6,30 +6,8 @@ class SphinxIndexationWorkerTest < ActiveSupport::TestCase
 
   include ActiveJob::TestHelper
 
-  test "callback" do
-    account = FactoryBot.create(:simple_account)
-    callback = ThinkingSphinx::RealTime.callback_for(:account)
-    assert callback
-
-    ThinkingSphinx::RealTime.expects(callback_for: callback)
-    callback.expects(:after_commit).with(Equals.new(account))
-    ThinkingSphinx::Test.rt_run do
-      perform_enqueued_jobs only: SphinxIndexationWorker do
-        SphinxIndexationWorker.perform_later(account.class, account.id)
-      end
-    end
-  end
-
-  test "only one index found per indexed model" do
-    worker = SphinxIndexationWorker.new
-    indices = ThinkingSphinx::Test.indexed_models.select do |model|
-      assert_match SizeMatcher.new(1), worker.send(:indices_for_model, model).to_a
-    end
-
-    assert_not_empty indices.map(&:name)
-  end
-
   test 'it does not raises if id does not exist' do
+    enable_search_jobs!
```

Comment:
Same, don't need to disable them after the test?

Reply:
Enabling and disabling should be automatic in most cases. This is of the rare cases that we need to alter it manually. It will be disabled again in next test case `setup`. And after `tierdown` actually the expected state is enabled :) And because we use mocha, it unstubs in teardown.

Reply:
If I stub `perform_later` as in the past though, this line will not be needed here. WDYT? Should we stub `#perform` or `::perform_later`?

Reply:
Honestly, no idea.

Reply:
I think this is fine. Disable means disable, not disable only background. Shouldn't make a big difference.